package.json

```diff
@@ -134,6 +137,7 @@
     "tslib": "^1.9.3",
     "tslint": "~5.11.0",
     "typescript": "~3.2.2",
+    "webpack": "^4.0.0",
```

Comment:
In a newly CLI-generated angular project, there are no `webpack` and `webpack-cli` list as a dependency. Maybe we can remove it, and rely on angular to maintain and the most compatible version?

Reply:
I think SSR needs it 

> "build:ssr": "ng run storefrontapp:server && webpack --config projects/ssr/webpack.server.config.js --progress --colors",

Reply:
Do you get an error if you remove it? It should still work, because we get `webpack` from angular.

Reply:
I get warnings if I remove it

Reply:
I believe that's fine, it should be installed by angular and put in `node_modules` to make it available for other packages